client/src/components/Comparison/Card/Card.jsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import card from './Card.module.css';
 
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
@qzyan What's this comment for?

Reply:
I will add a state to this component later. Now just disable the lint warning for this  stateless-function check